### PR TITLE
General Cleanup

### PR DESCRIPTION
**Pull request by ToasterUwU**
Cleaning up formatting issues, misspells, and other things my perfectionist brain cant leave standing.

### CODE REVIEW

**Comment by ToasterUwU** [comment]
The last commit was mainly for the sake of the linebreaks not displaying at all on Github ( Needs an empty line to actually line break )

**Comment by ToasterUwU** [comment]
Ran yapf ( that already was installed apparently ), did it with `yapf -vv -r -i .`, just so you know how I did it in case that matters and I didn't realize.

File: benchmark/create_random_file.py [hunk 1]
```diff
@@ -44,23 +44,30 @@ def main():
     parser.add_argument(
         "size",
         type=parse_size,
-        help="size in KiB/MiB/GiB, specify unit with a single suffix letter K/M/G, for example 256k for 256 KiB")
+        help="size in KiB/MiB/GiB, specify unit with a single suffix letter K/M/G, for example 256k for 256 KiB",
```
Comment (rsnitsch) [inline benchmark/create_random_file.py:47]: Why did you add the `,` in the end (right before the closing parenthesis)?

File: benchmark/create_random_file.py [hunk 2]
```diff
@@ -44,23 +44,30 @@ def main():
...
     parser.add_argument("--overwrite", action="store_true", help="Overwrite existing file.")
 
     args = parser.parse_args()
 
     if os.path.isfile(args.path) and not args.overwrite:
         if os.path.getsize(args.path) != args.size:
-            print("ERROR: Destination file already exists BUT DOES NOT HAVE THE CORRECT SIZE", file=sys.stderr)
+            print(
+                "ERROR: Destination file already exists BUT DOES NOT HAVE THE CORRECT SIZE",
+                file=sys.stderr,
```
Comment (rsnitsch) [inline benchmark/create_random_file.py:58]: Same here and right below. Why the comma?

File: benchmark/create_random_folder.py [hunk 3]
```diff
@@ -34,11 +34,15 @@ def main():
...
-        "minimum file size in KiB/MiB/GiB, specify unit with a single suffix letter K/M/G, for example 256k for 256 KiB"
+        "minimum file size in KiB/MiB/GiB, specify unit with a single suffix letter K/M/G, for example 256k for 256 KiB",
     )
     parser.add_argument("max_file_size", type=parse_size, help="maximum file size")
     parser.add_argument("--seed", type=int, default=0, help="Set seed for the random number generator.")
-    parser.add_argument("--no-cachedir-tag", action="store_true", help="Do not generate CACHEDIR.TAG file")
+    parser.add_argument(
+        "--no-cachedir-tag",
+        action="store_true",
+        help="Do not generate CACHEDIR.TAG file",
```
Comment (rsnitsch) [inline benchmark/create_random_folder.py:44]: Another comma.

File: benchmark/create_random_file.py (hunk 1 above)
Comment (ToasterUwU) [reply benchmark/create_random_file.py:47]: My Formatter does that automatically, not exactly sure why, don't really like it either, but it seems to be like that. 
And the one you use didn't remove it, so I assume its a PEP thing, but I never looked into it that much.

If you want me to remove those commas, let me know.

Comment (rsnitsch) [reply benchmark/create_random_file.py:47]: Yeah I think for consistency it's better to remove these commas. Unless you can find out the reason for the commas to be there. Maybe your formatter had good to reason to add them. But at the moment I don't see it.

Comment (ToasterUwU) [reply benchmark/create_random_file.py:47]: Ok, i did a little research.

Black, the formatter i use, adds and keeps these extra commas in situations where it isn't 100% sure that no comma will work as well. So if i ever save the file while its invalid or needs that comma to exist, it will add it, then if i save the file again later and its not actually needed anymore, it will keep it because "I put it there manually and clearly wanted it to stay".

So there is a reason for them existing, not a good one tho. I will remove them.

File: benchmark/plot_benchmark_results.py [hunk 4]
```diff
@@ -58,20 +64,24 @@ def main():
         else:
             df = pd.concat([df, df_file])
 
+    if df == None:
```
Comment (rsnitsch) [inline benchmark/plot_benchmark_results.py:67]: Comparing to `None` should always be done with `is` instead of `==`.

Comment (rsnitsch) [inline benchmark/plot_benchmark_results.py:67]: Comparing to `None` should always be done with `is` instead of `==`.

**Comment by rsnitsch** [comment]
Perfection. Thank you very much!
